### ISSUE TITLE
Accessibility evaluation of Benchmarks, Workshop, and Tools pages

### ISSUE THREAD

**Issue by agoslen** [DLFMetadataAssessment/Sandbox#340]
Please use the [WAVE tool](https://wave.webaim.org/) to do an initial accessibility evaluation of the following pages:

- [Metadata Quality Benchmarks](https://dlfmetadataassessment.github.io/Sandbox/projects/benchmarks/)
- [Metadata Analysis Workshop](https://dlfmetadataassessment.github.io/Sandbox/projects/metadata-workshop/)
- [Tools](https://dlfmetadataassessment.github.io/Sandbox/projects/tools/)

Summarize findings in the comments on this issue.

**Comment by sgolds15** [comment]
[Metadata Quality Benchmarks](https://dlfmetadataassessment.github.io/Sandbox/projects/benchmarks/): Contrast error; alerts for PDF/Excel file attachments, which may or may not be accessible.

[Metadata Analysis Workshop](https://dlfmetadataassessment.github.io/Sandbox/projects/metadata-workshop/): Contrast error; nondescriptive “here” link text.

[Tools](https://dlfmetadataassessment.github.io/Sandbox/projects/tools/): Contrast error; nondescriptive “here” link text; select element does not have an associated label.
